### DISCUSSION TITLE
第一次登陆，使用默认密码和账号，提示密码或账户错误。使用的版本是2.1.2 第一次登陆，使用默认密码和账号，提示密码或账户错误。使用的版本是2.1.2 When logging in for the first time, use the default password and account, and it will prompt that the password or account is incorrect. The version used is 2.1.2

### DISCUSSION THREAD

**Discussion by RichardMJT** [apache/streampark#3431] [Q&A]
<img width="1242" alt="捕获" src="https://github.com/apache/incubator-streampark/assets/51042944/409a9d54-857a-41d5-8b1b-830fb1f9266d">

**Comment by fansofstark** [comment]
请问解决了吗？？目前我也遇到了相同的问题

**Comment by caicancai** [comment]
cc @wolfboys @wangsizhu0504

**Comment by fengyudong1996** [comment]
这个问题在于你漏掉了一些步骤，可能是对数据库的一个初始化，初始化分为两步一步是通过schame文件创建数据库和表结构，另外还有一些初始数据插入，需要source两个文件。
